File: src/main/scala/me/maxih/aquaticcommandstestplugin/AquaticCommandsTestPlugin.scala
package me.maxih.aquaticcommandstestplugin

import me.maxih.aquaticcommands.framework.Commands
import org.bukkit.plugin.java.JavaPlugin

/**
  * Created by Maxi H. on 13.09.2018
  */
class AquaticCommandsTestPlugin extends JavaPlugin {
  override def onEnable(): Unit = {
    Commands.register(new PingCommand, new NukeCommand)
  }
}

File: src/main/java/me/maxih/aquaticcommandstestplugin/AquaticCommandsTestPlugin.java
package me.maxih.aquaticcommandstestplugin;

import me.maxih.aquaticcommands.framework.Commands;
import org.bukkit.plugin.java.JavaPlugin;

public class AquaticCommandsTestPlugin extends JavaPlugin {

    @Override
    public void onEnable() {
        Commands.register(ScalaUtil.varargs(new PingCommand(), new NukeCommand()));
    }

}
